lib/bkblz/v1/models.rb

```diff
@@ -6,25 +6,25 @@ module Model
       Bucket = Model.define :account_id, :bucket_id, :bucket_name, :bucket_type
 
       # Returned by list_file_versions
-      File = Model.define *[
+      File = Model.define(*[
```

Comment:
I'd prefer to just remove the splat and call Model.define like other calls in the file (i.e. with just the argument list)

I don't really know why I did this to begin with

same comment for lines below